go.mod

```diff
@@ -108,3 +118,5 @@ replace k8s.io/client-go v12.0.0+incompatible => k8s.io/client-go v0.21.4
 // Replace memberlist with our fork which includes some fixes that haven't been
 // merged upstream yet.
 replace github.com/hashicorp/memberlist => github.com/grafana/memberlist v0.3.1-0.20220708130638-bd88e10a3d91
+
+replace github.com/weaveworks/common => ../common
```

Comment:
replace in purpose, wait [pr in common](https://github.com/weaveworks/common/pull/291) to be merged